WordPress/Classes/Services/AccountSettingsService.swift

```diff
@@ -196,7 +196,16 @@ class AccountSettingsService {
         guard let results = (try? context.fetch(request)) as? [ManagedAccountSettings] else {
             return nil
         }
-        return results.first
+
+        guard let account = results.first else {
+            return nil
+        }
+
+        account.displayName = account.displayName.stringByDecodingXMLCharacters()
+        account.firstName = account.firstName.stringByDecodingXMLCharacters()
+        account.lastName = account.lastName.stringByDecodingXMLCharacters()
```

Comment:
This one feels a bit odd, since the method is expected to *load* from CoreData, but instead, it's also modifying few properties. Although this particular scenario is okay, i've got first hand experience debugging issues related to this practise, for hoooours and hoooours.

How about if, instead:

### Alternative A
. We add properties, like the one you've used below (`authorForDisplay`): **firstNameForDisplay** / **lastNameForDisplay** ... and, oh well, **displayNameForDisplay**?

### Alternative B
Probably cleaner and easier: maybe just do the conversion in the AccountSettings initialization?